portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java

```diff
@@ -35,21 +35,22 @@
   private static DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
 
   /**
-  * This Method handles GET requests in order to display all of the comments that are stored 
-  * in the Comments kind of the Google Cloud Datastore.
-  * <p>
-  * 
-  * @param  request  The <code>HttpServletRequest</code> for the GET request.
-  * @param  response The <code>HttpServletResponse</code> for the GET request.
-  * @return None. The Servlet writes to the /comment-data page which JavaScript then fetches 
-  *         in order to serve the comments to the UI. 
-  */
+   * This Method handles GET requests in order to display all of the comments that are stored in the
+   * Comments kind of the Google Cloud Datastore.
+   *
+   * <p>
```

Comment:
What is this tag for? Does it need to be closed?

Reply:
I was trying to follow this [doc](https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html) for styling and they used the paragraph tag for a line break. Is this the correct usage or no?

Reply:
After reading the comments section of the REAL Js style guide I realize they use Markdown and it is slightly different from the JavaDoc format. This was fixed in commit .

Reply:
Also corrected JavaDoc comments by removing unnecessary <p> tags.